### PR TITLE
[WFLY-13640] Reactive Messaging 1.0 with Kafka Support

### PR DESCRIPTION
**Pull request by kabir**
https://issues.redhat.com/browse/WFLY-13640
wildfly/wildfly-proposals#320
wildfly/wildfly-capabilities#48
wildfly/boms#98

Old version of PR is at https://github.com/wildfly/wildfly/pull/13783. This was pretty thoroughly reviewed. Apart from upgrades, the code changes are minimal

### CODE REVIEW

**Comment by kabir** [comment]
Unrelated failure `CertificateRolesLoginModuleTestCase.testDatabaseCertLoginModule`

**Comment by kabir** [comment]
Seems r

**Review by bstansberry — APPROVED** [review]

File: microprofile/galleon-common/pom.xml [hunk 1]
```diff
@@ -338,6 +360,138 @@
...
+        <dependency>
+            <groupId>io.smallrye.reactive</groupId>
+            <artifactId>smallrye-reactive-converter-api</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```
Comment (rhusar) [inline microprofile/galleon-common/pom.xml:427]: Nope! We already have this dependency in pom.xml... Let me fix this.

Comment (rhusar) [reply microprofile/galleon-common/pom.xml:427]: Same for the modules..

Comment (rhusar) [reply microprofile/galleon-common/pom.xml:427]: Opened https://issues.redhat.com/browse/WFLY-14480

Comment (rhusar) [reply microprofile/galleon-common/pom.xml:427]: PR is up fixing the version + duplicate declaration + jboss module: https://github.com/wildfly/wildfly/pull/14038

**Pull request merged**